microprofile/telemetry-smallrye/extension/pom.xml

```diff
@@ -146,6 +146,14 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wildfly</groupId>
```

Comment:
Minor nit. Elsewhere the `${project.groupId}` is used so it should likely be used here too.

Reply:
Use ${full.maven.groupId}.  Always use that or ${ee.maven.groupId} for artifacts coming out of this project's build. Which depends on whether the artifact is provisioned via wildfly or wildfly-ee feature pack.

Reply:
Please apply my comment ^^^ throughout the PR.